src/backends/tensorrt/plan_backend.h

```diff
@@ -232,8 +232,9 @@ class PlanBackend : public InferenceBackend {
     // Assume that the lifetime of the payload is extended until the completion
     // callback is called
     SyncQueue<std::tuple<
-        InferenceBackend*, size_t, const uint64_t, const size_t,
-        std::vector<std::unique_ptr<InferenceRequest>>*,
+        InferenceBackend*, size_t, const uint64_t, const uint64_t,
+        const uint64_t, const size_t,
+        std::shared_ptr<std::vector<std::unique_ptr<InferenceRequest>>>,
```

Comment:
use in-place or unique_ptr, not shared_ptr.

Reply:
Done